src/AlertsAggregator.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
+from datetime import datetime
+from models import AlertCountPerDay
+from Errors import WrongSettlementException
+from datetime import time, datetime, timedelta
+import requests
+GEONAMES_URL = "http://api.geonames.org/searchJSON"
+
+
+class AlertsAggregator:
+    def __init__(self, df):
+        self.df = df
+
+    # This func get dataframe and return a set of all the settlement in df
+    def return_data_list(self):
```

Comment:
very bad name for this function
what is data?? very generic word.
be more spesific when you name functions so it would be 100% clear from their name- what do they do!